antiweb.py

```diff
@@ -1465,18 +1465,156 @@ def filter_output(self, lines):
                 #remove comment chars in document lines
                 stext = l.text.lstrip()
 
-                if stext == '/*' or stext == "*/":
+                if stext == "/*" or stext == "*/":
                     #remove /* and */ from documentation lines
                     #see the l.text.lstrip()! if the lines ends with a white space
                     #the quotes will be kept! This is feature, to force the quotes
                     #in the output
                     continue
                 
-                if stext.startswith("//") and not stext.startswith("#####"):
-                    #remove comments but not chapters
+                if stext.startswith("//"):
                     l.text = l.indented(stext[2:])
                             
             yield l
+#edoc
+
+#@cstart(CSharpReader)
+class CSharpReader(CReader):
+    #@start(CSharpReader doc)
+    #CSharpReader
+    #=============
+    """
+    .. py:class:: CSharpReader
+
+       A reader for C# code. This class inherits :py:class:`CReader`.
+    """
+    
+    #@indent 3
+    #@include(CSharpReader)
+    #@include(CSharpReader.strip_tags doc)
+    #@include(CSharpReader.get_attribute_text doc)
+    #@include(CSharpReader.filter_output doc)
+    #@(CSharpReader doc)
+    
+    def _accept_token(self, token):
+        return super(CSharpReader, self)._accept_token(token)
```

Comment:
- So this is the call to the base method.
- It doesn't seem to do anything special
- Why is it needed then?